Automattic/sensei
# Add landing page patterns

Implements https://github.com/Automattic/sensei/issues/6053

### Changes proposed in this Pull Request

* Added 2 landing page patterns, one with a list view for the course list and another with a grid view as per design.

### Testing instructions

- Make sure to have the latest trunk of Course theme
- Create a new page
- Select a full-width template with no title
- Now from the inserter, select Sensei LMS from the dropdown
- Check that two new patterns are appearing, Landing Page and Landing Page - Grid
- Select any of them, save and check that they look as expected in the frontend

### Screenshot / Video
List-
![Screen Shot 2022-11-24 at 6 10 51 PM](https://user-images.githubusercontent.com/6820724/203783653-902afda2-ec4d-45b6-aa61-bd8db4a76d0a.png)

Grid-
![Screen Shot 2022-11-24 at 6 09 34 PM](https://user-images.githubusercontent.com/6820724/203783706-d1e28e17-4b50-437c-8030-1eb6fd1b218f.png)
